src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -89,7 +89,8 @@ public static void validateOptions(Options options) {
 
   }
 
-  private static final String VARIANT_FIELDS = "variants(start,calls(genotype,callSetName))";
+  private static final String VARIANT_FIELDS =
+      "variants(alternateBases,calls(callSetName,genotype),end,referenceBases,start)";
```

Comment:
@deflaux My code example is common in all introductory books to Java 8 - there is no intellectual property here unless we need to world to sign a CLA for all Java code, which does not make sense.  Based on the Google CLA I can write before each of my code comment **"Not a Contribution"** as noted on the [CLA website here](https://cla.developers.google.com/about/google-individual):

*"Contribution" shall mean any original work of authorship, including any modifications or additions to an existing work, that is intentionally submitted by You to Google for inclusion in, or documentation of, any of the products owned or managed by Google (the "Work"). For the purposes of this definition, "submitted" means any form of electronic, verbal, or written communication sent to Google or its representatives, including but not limited to communication on electronic mailing lists, source code control systems, and issue tracking systems that are managed by, or on behalf of, Google for the purpose of discussing and improving the Work, but excluding communication that is conspicuously marked or otherwise designated in writing by You as "Not a Contribution."*